BackofficeBundle/Controller/AreaController.php

```diff
@@ -75,18 +76,19 @@ public function templateFormAction(Request $request, $templateId, $areaId)
     }
 
     /**
-     * @param Request            $request
-     * @param string             $actionUrl
-     * @param AreaInterface      $area
-     * @param NodeInterface|null $node
+     * @param Request                $request
+     * @param string                 $actionUrl
+     * @param AreaInterface          $area
+     * @param TemplateInterface|null $template
      *
      * @return Response
      */
-    protected function generateForm(Request $request, $actionUrl, $area, NodeInterface $node = null)
+    protected function generateForm(Request $request, $actionUrl, $area, TemplateInterface $template = null)
```

Comment:
aren't you suppressing a functionnality ?

the last parameter should be a AreaContainerInterface (template or node) or StatusableInterface